pkg/terraform/terraform.go

```diff
@@ -102,22 +104,146 @@ type TerraformResponse struct {
 
 // TerraformBackendConfig represents the terraform backend config block.
 type TerraformBackendConfig struct {
+	// GCSBucket is the GCS Bucket used in a terraform backend config block.
 	GCSBucket *string `hcl:"bucket,attr"`
-	Prefix    *string `hcl:"prefix,attr"`
+	// Prefix is the GCS Bucket prefix used in a terraform backend config block.
+	Prefix *string `hcl:"prefix,attr"`
 }
 
+// // TerraformModule represents the terraform module block.
```

Comment:
This commented block looks like maybe you intended to remove it?

Reply:
Done